printrun/gui/graph.py

```diff
@@ -207,9 +207,8 @@ def _calculate_spacing(self):
             return 10 ** (exponent + 1)
 
     def drawtemperature(self, dc, gc, temperature_list,
-                        text, text_xoffset, r, g, b, a):
-        color = self.timer.IsRunning() and (r, g, b, a) or [128] * 4
```

Comment:
This line currently sets all graph colors to grey if the timer is not running. That functionality is lost with this PR but I think this is fine because I'm not sure why we had it in the first place.

Reply:
Yeah, I removed it since I never saw it happening in practice and determined that it's probably not useful

Can of course add it back too if needed though

Reply:
Yeah, let's drop it for now.

Reply:
I would leave the gray color intact.
I consider it a safety feature. The graph is indicator about burning hot temperatures.
If for some reasons the graph does not correspond to reality, a fire can occur or someone get burned.
What can go wrong?
For example an exception before Start(). The timer is strangely re-Start()-ed on every event, although it's default parameter is oneShot=TIMER_CONTINUOUS.
Another scenario is enabling/disabling the graph control in Settings. There are glitches with the addition/removal of controls in the code base - e.g. "C++ object has been deleted".
Even if the current code does not throw exceptions/have bugs, how can one be sure that some developer (printrun or wxwidgets) won't add any in the future?